### PR TITLE
DLS-9376 - Added captions and moved h2

### PR DESCRIPTION
**Pull request by Timidigital**
# JIRA Story or Github Number - descriptive title

**Bug fix / New feature** (delete as appropriate)

Please include a summary / description of the change and which issue it fixes.  Include any relevant user needs, context or links to other PRs related to this PR (eg. acceptance tests, environment config).

## Checklist

 - [ ]  I've made every effort to commit high quality, clean code and I have executed relevant static analyses to be sure
 - [ ]  I've included appropriate tests with any code I've added (Unit, Integration, Acceptance etc.)
 - [ ]  I've executed the acceptance test pack locally to ensure there are no functional regressions
 - [ ]  I've added my code using logical, atomic commits, squashing as appropriate - including the JIRA issue number in the commit message
 - [ ]  I've run a dependency check to ensure all dependencies are up to date

### CODE REVIEW

File: app/views/playComponents/summaryPartialTaxableGain.scala.html [hunk 1]
```diff
@@ -20,10 +20,11 @@
 
 <div id="yourTaxableGain">
 
+    <h3 class="govuk-heading-m">@Messages("calc.summaryPartial.yourTaxableGain")</h3>
```
Comment (nubz) [inline app/views/playComponents/summaryPartialTaxableGain.scala.html:23]: this should be protected with the same condition as it was when it was in the table `@if(gain > 0) {`

File: test/views/playComponents/SummaryPartialTaxToPayViewSpec.scala [hunk 2]
```diff
@@ -145,7 +158,10 @@ class SummaryPartialTaxToPayViewSpec extends CommonPlaySpec with WithCommonFakeA
       lazy val doc = Jsoup.parse(view.body)
 
       s"does not have the text ${messages.yourTaxRate}" in {
-        doc.select("h3").isEmpty shouldBe true
+        doc.select("h3").isEmpty shouldBe false
```
Comment (nubz) [inline test/views/playComponents/SummaryPartialTaxToPayViewSpec.scala:161]: this should still be the same test and equate to `true` which would pass if the h3 was conditional (other comment I made)

**Review by nubz — CHANGES_REQUESTED** [review]

File: test/views/playComponents/SummaryPartialTaxToPayViewSpec.scala (hunk 2 above)
Comment (Timidigital) [reply test/views/playComponents/SummaryPartialTaxToPayViewSpec.scala:161]: updated on [da47d1](https://github.com/hmrc/cgt-calculator-non-resident-frontend/pull/268/commits/3da47d1e6af12b9c9be923b17eaa7fbfaed2f2ca)

File: app/views/playComponents/summaryPartialTaxableGain.scala.html (hunk 1 above)
Comment (Timidigital) [reply app/views/playComponents/summaryPartialTaxableGain.scala.html]: updated on [da47d1](https://github.com/hmrc/cgt-calculator-non-resident-frontend/pull/268/commits/3da47d1e6af12b9c9be923b17eaa7fbfaed2f2ca)

File: app/views/playComponents/summaryPartialTaxToPay.scala.html [hunk 3]
```diff
@@ -19,12 +19,17 @@
 @(taxToPay: BigDecimal, gainAtBandOne: BigDecimal, gainAtBandTwo: BigDecimal, taxAtBandOne: BigDecimal, taxAtBandTwo: BigDecimal, taxRateOne: Int, taxRateTwo: Int)(implicit request: Request[_], messages: Messages, messagesApi: Option[MessagesApi] = None)
 
 <div id="yourTaxRate">
-    <table class="govuk-table">
+    @if(taxToPay > 0) {
+        <h3 class="govuk-heading-m">@Messages("calc.summaryPartial.yourTaxRate")</h3>
+    }
+
+    <p class="govuk-body">@Messages("calc.summaryPartial.incomeBandInfo")</p>
```
Comment (nubz) [inline app/views/playComponents/summaryPartialTaxToPay.scala.html:26]: this should also be inside the conditional (as it was in the table)

**Review by nubz — CHANGES_REQUESTED** [review]

**Review by nubz — APPROVED** [review]

**Pull request merged**